count-things/README.md

```diff
@@ -0,0 +1,10 @@
+# Count Things
+
+> "when in doubt, count"
+
+use: `node ./index.js corpus=name-of-corpus`
+
+The script will:
+
+1. analyze a folder of code stored in `../.corpora/name-of-corpus`
```

Comment:
if there's a possibility to just analyze one folder, we might want to be more specific with the terms. "Corpus" usually refers to a principled collection of things (in linguistics, texts, and here, code), which wouldn't normally be one project, meaning a single repository.

We could probably use the term in the case of a number of projects, but to avoid misunderstandings, it might be better to be very explicit here that by "folder" we mean "a directory with a bunch of subdirectories"...or...separate out analysis of multiple directores from analysis of a single git repository.

Reply:
I was thinking that the `/.corpora` folder would store many corpora with whatever name you wanted to give each one.  I did this locally to test, making different ones to test different counts.  Each folder inside `/.corpora` is treated as a separate thing to analyze by this script and it's up to the user to decide what goes in each one.

Is that what you're saying?  I don't think we're so far apart

Reply:
ah wait, you're just talking about the wording in the README and not the folder naming?